### ISSUE TITLE
Reports Edit.GoToFindCombo (Ctrl+D) repeatedly

### ISSUE THREAD

**Issue by rprouse** [madskristensen/ShowTheShortcut#3]
### Installed product versions
- Visual Studio: 2015 Professional Update 2
- This extension: 1.4.19

### Description
The extension is reporting Edit.GoToFindCombo (Ctrl+D) about once per second in the output window.

### Steps to recreate
1. Install extension
2. Start Visual Studio
3. Open the output window

### Current behavior

No solution open (or with solution open), it is repeatedly reporting. This makes the extension unusable because real events are quickly replaced in the status bar with this event. Happens even when another app has focus.

```
2016-05-04 12:05:26 PM: Edit.GoToFindCombo (Ctrl+D)
2016-05-04 12:05:27 PM: Edit.GoToFindCombo (Ctrl+D)
2016-05-04 12:05:27 PM: Edit.GoToFindCombo (Ctrl+D)
2016-05-04 12:05:28 PM: Edit.GoToFindCombo (Ctrl+D)
2016-05-04 12:05:29 PM: Edit.GoToFindCombo (Ctrl+D)
2016-05-04 12:05:31 PM: File.OpenProject (Ctrl+Shift+O)
2016-05-04 12:05:31 PM: Edit.GoToFindCombo (Ctrl+D)
2016-05-04 12:05:32 PM: Edit.GoToFindCombo (Ctrl+D)
2016-05-04 12:05:32 PM: Edit.GoToFindCombo (Ctrl+D)
2016-05-04 12:05:33 PM: Edit.GoToFindCombo (Ctrl+D)
```

### Workaround
If I remove the Standard Toolbar (right click in Toolbars and disable Standard) it stops happening. Customizing the Standard Toolbar and removing the Find Combo also stops it from happening.

![image](https://cloud.githubusercontent.com/assets/493828/15020512/51d2be06-11f0-11e6-96ed-67deeb604401.png)

**Comment by madskristensen** [comment]
Thanks for the bug report. Please help test the fix in the latest [CI build](http://vsixgallery.com/extension/9da28329-f9d5-4f18-91c3-d3285b103d1a/)

**Comment by rprouse** [comment]
Thanks for the quick fix @madskristensen.

Confirmed fixed in 1.4.20
